course/models.py

```diff
@@ -131,7 +137,8 @@ class Enrollment(models.Model):
     timestamp = models.DateTimeField(auto_now_add=True)
     personal_code = models.CharField(max_length=10, blank=True, default='')
     selected_group = models.ForeignKey(StudentGroup, blank=True, null=True, default=None, on_delete=models.SET_NULL)
-
+    anon_name = models.CharField(max_length=50, blank=True, default='')
+    anon_id = models.CharField(max_length=50, null=True, unique=True)
```

Comment:
nyt, kaikkien kurssien yli `anon_id` pitäis olla uniikki, mutta me tarvitaan vain kyseisen kurssi-instanssin yli. Sen saa aikaan tälläsellä:
https://docs.djangoproject.com/en/2.0/ref/models/options/#unique-together

Reply:
Varmaan myös hyvä luoda `index_together`, koska nää haetaan aina instanssi + id parilla.